### DISCUSSION TITLE
Awus1900 Issues

### DISCUSSION THREAD

**Discussion by BenjaminPatrickL** [morrownr/8814au#101] [Q&A]
I just received my adapter and tested it on my friends laptop.

I installed the realtek driver utility for windows & rebooted the pc (no go) 
The adapter cant even be found.

I've also tried to install through Linux with no prevail (not showing in USB Devices Virtualbox)

My questions are:

Does the Adapter only function on USB 3.0?

**Comment by morrownr** [comment]
Hi @BenjaminPatrickL 

> Does the Adapter only function on USB 3.0?

It will work in USB2 mode as well as USB3.

> I've also tried to install through Linux with no prevail (not showing in USB Devices Virtualbox)

Installing in a VM is something you need to know how to do. There are guides. I have a link to a guide in the FAQ.

> I installed the realtek driver utility for windows & rebooted the pc (no go)
The adapter cant even be found.

Can't help with this. I don't use Windows.

The best advice I can give you is that if you are going to be working with Linux, send this adapter back and get your money back.  Below are some links that may help you:

https://github.com/morrownr/USB-WiFi

The above link is the Main Menu to this site. I recommend reading menu items 1, 2 and 5. That should help explain a few things. I recommend users stay away from adapters that use this 8814au chipset as this driver is old and is missing many features. Realtek has simply failed to support their product. I keep this driver here so those that have a 8814au based adapter can do what they can with it but if you just bought and can get your money back, I recommend you so.

If you have questions, post in `Issues` at the Main Menu.

Regards
